app/ApiClients/Oxidized.php

```diff
@@ -36,6 +36,7 @@ class Oxidized extends BaseApi
 
     public function __construct()
     {
+        parent::__construct(90);
```

Comment:
Think it's better we do as the option below, set the timeout value to what you want here, and set 3s in base

Reply:
Hi, I'm not sure to understand if you agree or request a change, and if so, what change? (set the timeout instead of calling the parent contructor?)

Reply:
@Jellyfrog is this what you asked for?

Reply:
Yes. And remove the other constructs ofc :)